wecs/panda3d/character.py

```diff
@@ -115,7 +123,6 @@ class UpdateCharacter(System):
     entity_filters = {
         'character': and_filter([
             CharacterController,
-            Model,
```

Comment:
Updating a character without a `Model` makes no sense, since without it, the character has no position in space.